WALLET.md

```diff
@@ -236,25 +236,23 @@ Now follow [the steps for iOS installation](#install-cocopods-bundler-and-downlo
 
 #### Install Java
 
-We need Java to be able to build and deploy the mobile app to Android devices. Android currently only builds correctly with Java 8. (Using OpenJDK because of [Oracle being Oracle][oracle being oracle]).
+We need Java to be able to build and deploy the mobile app to Android devices. Android currently builds with Java 11.
 
 ##### MacOS
 
 Install by running the following:
 
 ```bash
-brew install cask
-brew tap homebrew/cask-versions
-brew install --cask homebrew/cask-versions/adoptopenjdk8
+brew install java11
```

Comment:
React Native [recommends OpenJDK zulu11](https://reactnative.dev/docs/environment-setup):
> We recommend installing the OpenJDK distribution called Azul Zulu using [Homebrew](http://brew.sh/). Run the following commands in a Terminal after installing Homebrew:
>
> ```
> brew tap homebrew/cask-versions
> brew install --cask zulu11
> ```
>
> The Zulu OpenJDK distribution offers JDKs for both Intel and M1 Macs. This will make sure your builds are faster on M1 Macs compared to using an Intel-based JDK.

Was there a problem using it?

Reply:
Nope, just tried this and it works fine. Updated the readme